src/ansys/fluent/core/launcher/fluent_container.py

```diff
@@ -64,6 +64,24 @@
 DEFAULT_CONTAINER_MOUNT_PATH = "/mnt/pyfluent"
 
 
+class FluentImageNameTagNotSpecified(ValueError):
+    """Raises exception if Fluent image name or image tag is not specified."""
+
+    pass
+
+
+class ServerInfoFileError(ValueError):
+    """Raises exception if server info file is not given properly."""
+
+    pass
+
+
+class LicenseServerNotSpecified(KeyError):
+    """Raises exception if license server is not specified."""
+
+    pass
+
+
```

Comment:
Same issue. Caller is providing the generalised message.

Reply:
We will update it in all places.

Reply:
Done. Thank you.